pkgs/build-support/build-fhs-userenv-bubblewrap/env.nix

```diff
@@ -32,7 +32,7 @@ let
 
   # list of packages which are installed for both x86 and x86_64 on x86_64
   # systems
-  multiPaths = multiPkgs pkgsi686Linux;
+  multiPaths = lib.optional isx86_64 (multiPkgs pkgsi686Linux);
```

Comment:
This is still quite a bit different from https://github.com/NixOS/nixpkgs/pull/132963, in which this will also remove the `32bit` versions of all the packages. where as the other will only exclude `glibc_multi`